installing/installing_oci/installing-oci-agent-based-installer.adoc

```diff
@@ -37,7 +37,7 @@ include::modules/creating-config-files-cluster-install-oci.adoc[leveloffset=+1]
 
 * xref:../../architecture/architecture-installation.adoc#installation-overview_architecture-installation[About {product-title} installation]
 * xref:../../installing/overview/installing-preparing.adoc#installing-preparing-selecting-cluster-type[Selecting a cluster installation type]
-* xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#about-the-agent-based-installer[Preparing to install with the Agent-based Installer]
+* xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#about-the-agent-based-installer_preparing-to-install-with-agent-based-installer[Preparing to install with the Agent-based Installer]
```

Comment:
This actually jumps down to the "About the Agent-based Installer" module. Should it instead link to the very top of the assembly, to match the link text of "Preparing to install with the Agent-based Installer"?

Reply:
Hmm good catch! I didn't add the link originally and don't know the intent, but I'm guessing this should point to the top of the assembly (which is directly above this linked section anyways)